packages/sdk/src/actor/behaviors/action.ts

```diff
@@ -1,86 +1,94 @@
-/*!
```

Comment:
Did the whitespace change in this file? Diff is failing.

Reply:
No the white space is unchanged.  This happened a couple of times since the file move that Steven had done.